_component_design/alert-bar.md

```diff
@@ -6,9 +6,11 @@ category: Components
 usage: >
   A notification bar extends fully across the top of an content container within a Helix interface in order to provide the user with feedback relating to the status of the item. It can be scoped across the entirity of the content area (page level scoping) or scoped to extend across a singular section of the content area (element level scoping).
 preview-image: preview-images/alert-bar.svg
-status: deprecated, in-progress
+status: in-progress
 resource: true
 last-modified: 2017-08-17
+helix-ui-css: true
```

Comment:
Long-term thought: We might be able to just query their site's source to find relevant components (and populate the status page with components that don't have a design page). Something to file away. Not sure how we'd do it yet, but you never know :)

Reply:
I agree.  I could see us harmonizing the names of our patterns so there's more of a 1:1 relationship -- at least when the code exists.  I also wonder if the helix-ui site could render an RSS feed we could consume, for things like this?

Reply:
Hmmm... that could work. There's a lot of plugins to do stuff like that for Jekyll, at least; I imagine whatever platform they're on currently has a similar system.